lib/themes-panel.coffee

```diff
@@ -147,6 +153,43 @@ class ThemesPanel extends View
     @activeUiTheme = @getActiveUiTheme()
     @activeSyntaxTheme = @getActiveSyntaxTheme()
     @populateThemeMenus()
+    @toggleActiveThemeButtons()
+    @handleActiveThemeButtons()
+
+  handleActiveThemeButtons: ->
```

Comment:
You mean something like `handleActiveThemeButtonEvents`? (`handleEvents` seems a bit too generic here)